monitoring/prometheus/ymls/local.yml

```diff
@@ -0,0 +1,62 @@
+  ###################################################
+  # monitor local (remote-dev) setups that use docker
+  ###################################################
+
+  # monitor discovery provider containers
+
+  - job_name: 'local-discovery-provider'
+    scheme: 'http'
+    metrics_path: '/prometheus_metrics'
+    static_configs:
+      - targets: ['host.docker.internal:5000']
+        labels:
+          host: 'host.docker.internal'
+          environment: 'remote-dev'
+          service: 'audius'
+          component: 'discovery-provider'
+
+  # monitor content node containers
```

Comment:
Added these here so that we could get unique `job_name`s missing from #3266.

Reply:
amazing